packages/storybook/jest.config.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  collectCoverageFrom: [
+    "**/src/**/*.{js,jsx}",
+    "!**/index.js",
```

Comment:
Excludes *all* `index.js` files from counting towards code coverage. Should be safe if we exclusively use them for re-exporting a component's internals?

Reply:
I support that. I'd like to see them just used for exporting anyway... Should we be testing all our exports?

Reply:
Lets leave them in, they're a part of the source after all. We should be testing our exports anyhow.

Reply:
Lets leave them in, they're a part of the source after all. We should be testing our exports anyhow.